web3/main.py

```diff
@@ -89,9 +89,11 @@ class Web3(object):
 
     # Encoding and Decoding
     toHex = staticmethod(to_hex)
-    toAscii = staticmethod(decode_hex)
+    toBytes = staticmethod(decode_hex)
+    toAscii = toBytes
```

Comment:
Thoughts on going ahead and marking this deprecated?

Reply:
I'm a fan, will do